app/assets/javascripts/components/activity/activity_handler.jsx

```diff
@@ -18,6 +18,10 @@ export const ActivityHandler = createReactClass({
     course: PropTypes.object,
   },
 
+  componentDidMount() {
+    document.title = `${I18n.t('activity.label')} - ${this.props.course.title}`;
+  },
```

Comment:
There are two tabs for this component, Recent Activity, and Possible Plagiarism. It would be useful to have different titles for each one of them instead of one title in the handler component. This goes for Articles and Students tabs also.

Reply:
Oh yeah I missed that. I guess I'll wait for Sage's input before continuing down this way of adding titles.